webapp/components/anomalies.html

```diff
@@ -0,0 +1,135 @@
+<link rel="import" href="../bower_components/polymer/lib/elements/dom-if.html">
+<link rel="import" href="../bower_components/polymer/lib/elements/dom-repeat.html">
+<link rel="import" href="runs.html">
+<link rel="import" href="test-file-results.html">
+<link rel="import" href="test-run.html">
+
+<dom-module id="wpt-anomalies">
+  <template>
+    <style>
+    </style>
+
+      <section class="caveat">
+        Data below are intended for web platform implementers and do not contain useful metrics for evaluation or comparison of web platform features. Also note that tested Edge and Safari are not pre-release versions (<a href="https://github.com/w3c/wptdashboard/issues/109">#109</a>, <a href="https://github.com/w3c/wptdashboard/issues/110">#110</a>).
+      </section>
+
+    <template is="dom-repeat" items="{{anomalyGroups}}" as="anomalyGroup">
+      <h2>Passing in [[anomalyGroup.passingBrowsers]] browsers</h2>
+      <template is="dom-repeat" items="{{anomalyGroup.tests}}" as="test">
+        <a href="[[test]]">[[test]]</a><br />
+      </template>
+    </template>
+  </template>
+
+  <script>
+    /**
+     * Component for viewing a list of anomalies in a group of TestRuns across
+     * multiple browsers, where anomalies are tests which are not interoperable
+     * (i.e. pass in some, but not all, browsers).
+     */
+    class WPTAnomalies extends window.Polymer.Element {
+      static get is() { return 'wpt-anomalies'; }
+
+      static get properties() {
+        return {
+          /**
+           * Metadata including the source url, runs, and times for the metrics
+           * computed for a set of test runs.
+           */
+          passRateMetadata: {
+            type: Object,
+          },
+          /**
+           * The processed contents of the metrics from the source url, once it
+           * has been fetched.
+           * Contains a map of [Pass rate] -> [Passing tests]
+           * where [Pass rate] is the number of browsers that a test passes in.
+           */
+          anomalyGroups: {
+            type: Object,
+          },
+          /**
+           * Browser for which to view browser-specific anomalies (optional).
+           */
+          browser: {
+            type: String,
+            value: ''
+          }
+        };
+      }
+
+      async connectedCallback() {
+        await super.connectedCallback();
+
+        // Fetch the results JSON for the TestRuns
+        const metrics = await this._fetchResults(this.passRateMetadata.url);
+
+        let anomalies = metrics.data;
+        if (!this.browser) {
+          anomalies = anomalies.filter(d => this._isTestFile(d.dir));
+        }
+        
+        const totalBrowsers = metrics.metadata.test_runs.length;
+        let matchesPassRate = !this.browser
+          ? data => data.pass_rates[passes] < 1 || !this._isTestFile(data.dir)
+          : data => data.num_other_failures + 1 < totalBrowsers;
+        let testPath = !this.browser
```

Comment:
`testPath` is unused. On line 98, `let testPath = ...` occurs and shadows this binding, but _this_ particular `testPath` is not used before that point.

Reply:
Done.